stable/filebeat/values.yaml

```diff
@@ -145,6 +145,17 @@ podSecurityPolicy:
 ## Dont forget to enable http on config.http.enabled (exposing filebeat stats)
 monitoring:
   enabled: true
+  serviceMonitor:
+    # When set true and if Prometheus Operator is installed then use a ServiceMonitor to configure scraping
+    enabled: true
+    # Set the namespace the ServiceMonitor should be deployed
+    namespace: monitoring
```

Comment:
Is this a common case where it's deployed to the monitoring namespace?  I have been having these deploy to whatever namespace the  chart is deployed into and I would prefer that to be the default?

Reply:
It may not be common to deploy them to a namespace called `monitoring` however I made this overridable in this way so that you can define the namespace that the ServiceMonitor should install into because the ServiceMonitor must be in the same namespace that the Prometheus Resources have been installed into (basically the same namespace as the Prometheus Operator). This is described here: https://github.com/coreos/prometheus-operator/blob/master/Documentation/user-guides/running-exporters.md#namespace-limitsthings-to-keep-in-mind

Should I set the default to something other than `monitoring`? Or maybe not set a default at all? Or maybe you have a completely different suggestion? :)

Reply:
Let's go with not setting it at all, but allowing people to override it.

Reply:
Pushed a new commit to reflect this 👍